taskcluster/kinds/client/kind.yml

```diff
@@ -80,7 +80,7 @@ tasks:
       command: >-
         eval $(python test/client-library-secrets.py) &&
         cd clients/client-py &&
-        TOXENV=py39 uvx tox
+        uv run pytest
   py310:
```

Comment:
Completely unrelated drive-by comment, but Taskgraph supports Github actions-like matrices you could use for this:
https://github.com/taskcluster/taskgraph/blob/main/taskcluster/kinds/test/kind.yml#L45

Reply:
Was toying around with this yesterday! https://github.com/taskcluster/taskcluster/pull/8018/files#diff-ee26b47dfb4e74101a5c3a3f8651c9327b5d03078a351e6e655edf27bcfa3b9a